extensions/historical-imagery/schema.json

```diff
@@ -63,41 +64,49 @@
           "type": "array",
           "allOf": [
             {
+              "type": "array",
```

Comment:
Is it necessary to add "array" at this level given it is already specified in line 64 above?

Reply:
Hmm, actually no. I'll check what else can be simplified.

Reply:
Resolved